lesson02/assignment/css/_card.scss

```diff
@@ -0,0 +1,9 @@
+.card{
+  background-color:white;
+  width:50em;
+  margin:25px;
```

Comment:
For responsive design, we don't want to use static sizing. Almost all rules should be sized with units of `%` or `rem`.